src/test/java/org/folio/readingroom/controller/PatronPermissionControllerTest.java

```diff
@@ -91,6 +97,124 @@ void testInvalidReadingRoomIdUpdatePatronPermission() throws Exception {
       .andExpect(status().is4xxClientError());
   }
 
+  @Test
+  void testSuccessGetPatronPermission() throws Exception {
+    removeReadingRoomIfExists();
+    ReadingRoom readingRoom = createReadingRoom(READING_ROOM_ID_FOR_PATRON_TEST, false);
+    readingRoom.setName("TestPatronRR");
+    var servicePoints = Set.of(createServicePoint(SERVICE_POINT_ID4, SERVICE_POINT_NAME3));
+    readingRoom.servicePoints(servicePoints);
+
+    this.mockMvc.perform(
+        post("/reading-room")
+          .content(asJsonString(readingRoom))
+          .headers(defaultHeaders())
+          .contentType(MediaType.APPLICATION_JSON)
+          .accept(MediaType.APPLICATION_JSON))
+      .andExpect(status().isCreated());
+
+    this.mockMvc.perform(
+        get("/reading-room-patron-permission/2205005b-ca51-4a04-87fd-938eefa8f6df")
+          .headers(defaultHeaders())
+          .contentType(MediaType.APPLICATION_JSON)
+          .accept(MediaType.APPLICATION_JSON))
+      .andExpect(status().is2xxSuccessful())
+      .andExpect(jsonPath("$[0].readingRoomId").value("7a5abc9f-f3d7-4856-b8d7-6712462ca009"))
+      .andExpect(jsonPath("$.[0].userId").value("2205005b-ca51-4a04-87fd-938eefa8f6df"))
+      .andExpect(jsonPath("$.[0].access").value(NOT_ALLOWED.getValue()))
+      .andExpect(jsonPath("$", hasSize(1)))
+      .andDo(print());
```

Comment:
I think this line can be removed from here and from line 215

Reply:
fixed